core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -45,54 +46,61 @@ class ConfigCommandTest extends Logging {
   private val zkConnect = "localhost:2181"
   private val dummyAdminZkClient = new DummyAdminZkClient(null)
 
-  @Test
```

Comment:
We don't need to add Kraft support here since ConfigCommandTest is just a unit test